config.py

```diff
@@ -678,6 +678,8 @@
 Passing a dictionary allows to assign a name to map a complex condition name
 (value) to a more legible one (value).
 
+This is a REQUIRED parameter in the configuration file. If left None it will raise an error.
```

Comment:
Keep the lines at 79 characters please :)
I've also changed the uppercase `REQUIRED` to bold lowercase.
```suggestion
This is a **required** parameter in the configuration file. If left as `None`,
it will raise an error.
```